domain/src/main/assets/coming_soon_topics.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Placeholder reminding this to be removed once I add a PR for doing this correctly.

Reply:
Yeah sure.

Reply:
#2303 for reference.

Reply:
Updated with the changes.

Reply:
This file is not a part of this PR anymore. Resolving this comment.